src/simplewallet/password_container.cpp

```diff
@@ -237,16 +266,16 @@ namespace tools
       }
       else if (ch == BACKSPACE)
       {
-        if (!m_password.empty())
+        if (!aPass.empty())
         {
-          m_password.back() = '\0';
-          m_password.resize(m_password.size() - 1);
+          aPass.back() = '\0';
+          aPass.resize(aPass.size() - 1);
           std::cout << "\b \b";
         }
       }
       else
       {
-        m_password.push_back(ch);
```

Comment:
m_password isn't used at all anymore, right ? It could go from the object if so.